games/TicTacToe/packages/app/src/App.tsx

```diff
@@ -0,0 +1,38 @@
+import { Container, Typography, Stack, CssBaseline, Box } from "@mui/material";
+import { Toaster } from "react-hot-toast";
+import { Board, ConnectionInfo, NewGameButton } from "./components";
+import { useGetGameState } from "./hooks";
+import { useAppContext } from "./components";
+
+function App() {
+  const { gameState } = useGetGameState();
+  const appContext = useAppContext();
+
+  return (
+    <>
+      <CssBaseline />
+      <Container>
+        <Stack alignItems="center" spacing={2} sx={{ marginTop: "32px" }}>
+          <Box
+            display="flex"
+            alignItems="center"
+            justifyContent="space-between"
+            width="100%"
+            flexWrap="wrap"
+          >
+            <Box width="300px"></Box>
+            <Typography align="center" variant="h3">
+              TicTacToe
+            </Typography>
+            <ConnectionInfo />
+          </Box>
+          {gameState === "Ended" && <NewGameButton />}
```

Comment:
The gameState was undefined for me when I first connected so this was blank, idk if I didn't set it up right?

Reply:
I'm pretty sure this happened bc your wallet was on the wrong network.  I'll come up with some sort of check to make sure the user is on the right network and if not give a warning/error